### DISCUSSION TITLE
Is it possible to view and generate API Load test results in table format

### DISCUSSION THREAD

**Discussion by rrnareshkumar** [artilleryio/artillery#2119] [Artillery]
Hi , Please suggest if it is possible to view and generate API load test results in table format .

currently I am generating report in json format , converting it into html file and using it but would like to understand if i can enable (if there are options) or customize to get API load test results/report in table format . 

For example : some thing like below in table format

APIs | Average   Response time | 90th Percentile   response time | Min response   time | Max response   time | Total Requests   Count | Passed (https   code 200) | Failes requests | Error% | Sent bytes | received bytes

API1 | 300 | 500 | 100 | 550 | 10 | 8 | 2 | 20% | 20kb | 200kb
API2 | 400 | 600 | 150 | 650 | 35 | 7 | 3 | 30% | 30kb | 300kb

**Comment by bernardobridge** [comment]
Hello 👋 ,

Currently, no other formats are supported. As per our documentation, the way to view the report [is JSON](https://www.artillery.io/docs/reference/cli/run#--output---create-a-json-report), and it can also be converted to HTML (https://www.artillery.io/docs/reference/cli/report) - although this is not actively maintained right now.

Soon we will be launching [Artillery Cloud](https://github.com/artilleryio/artillery/discussions/1948) which is a much better way to visualise and analyse results (currently in beta).

If you want to display results in other formats, you would have to (for example) [write your own plugin](https://www.artillery.io/blog/extend-artillery-by-creating-your-own-plugins) which can hook to the `stats` event and convert it to other formats you may need. Or you can write a script to interact with the JSON report at the end and display the data in other ways you may need.
